### PR TITLE
Add document create

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do?

Adds `document:create` method.

### Usage

`kourou document:create index collection --body '{ "toto": "titi" }'`

```
USAGE
  $ kourou document:create INDEX COLLECTION

ARGUMENTS
  INDEX       Index name
  COLLECTION  Collection name

OPTIONS
  -h, --host=host      [default: localhost] Kuzzle server host
  -p, --port=port      [default: 7512] Kuzzle server port
  --body=body          (required) Request body in JSON format.
  --help               show CLI help
  --id=id              Optionnal document ID
  --password=password  Kuzzle user password
  --replace            Replace the document if it already exists.
  --ssl                Use SSL to connect to Kuzzle
  --username=username  [default: anonymous] Kuzzle username (local strategy)
```

### Other changes

 - `query` command arg are now separated by `=` instead of `:` (eg: `kourou query index:create --arg index=nyc-open-data`)
 - use api key methods from the JS SDK
 - add logging when connecting to Kuzzle

### Boyscout

 - improve existing functional tests

### CODE REVIEW

File: src/commands/document/create.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+export default class DocumentCreate extends Kommand {
+  static description = 'Creates a document'
+
+  static flags = {
+    body: flags.string({
+      description: 'Request body in JSON format.',
+      required: true
+    }),
+    id: flags.string({
+      description: 'Optionnal document ID'
```
Comment (Yoann-Abbes) [inline src/commands/document/create.ts]:
```suggestion
      description: 'Optional document ID'
```
:feelsgood:

File: src/commands/document/create.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    }),
+    replace: flags.boolean({
+      description: 'Replace the document if it already exists.'
```
Comment (Yoann-Abbes) [inline src/commands/document/create.ts]:
```suggestion
      description: 'Replaces the document if it already exists.'
```
:sob:

File: README.md [hunk 3]
```diff
@@ -198,6 +199,32 @@ OPTIONS
...
+  COLLECTION  Collection name
+
+OPTIONS
+  -h, --host=host      [default: localhost] Kuzzle server host
+  -p, --port=port      [default: 7512] Kuzzle server port
+  --body=body          (required) Request body in JSON format.
+  --help               show CLI help
+  --id=id              Optionnal document ID
+  --password=password  Kuzzle user password
+  --replace            Replace the document if it already exists.
```
Comment (scottinet) [inline README.md]:
```suggestion
  --body=body          (required) Request body in JSON format
  --help               show CLI help
  --id=id              Optional document ID
  --password=password  Kuzzle user password
  --replace            Replace the document if it already exists
```

File: src/commands/document/create.ts [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  static flags = {
+    body: flags.string({
+      description: 'Request body in JSON format.',
+      required: true
+    }),
+    id: flags.string({
+      description: 'Optional document ID'
+    }),
+    replace: flags.boolean({
+      description: 'Replaces the document if it already exists.'
```
Comment (scottinet) [inline src/commands/document/create.ts]:
```suggestion
      description: 'Request body in JSON format',
      required: true
    }),
    id: flags.string({
      description: 'Optional document ID'
    }),
    replace: flags.boolean({
      description: 'Replaces the document if it already exists'
```

**Pull request merged**